src/applications/financial-status-report/components/householdExpenses/InstallmentContractSummary.jsx

```diff
@@ -32,10 +33,6 @@ const InstallmentContractSummary = ({
   const formData = useSelector(state => state.form.data);
   const { installmentContracts = [] } = formData;
 
-  useEffect(() => {
-    clearJobIndex();
-  }, []);
-
```

Comment:
Another place where I removed useEffect because it didn't seem to be related to the installmentContracts component, let me know if I need to add it back in